src/scss/skin-legacy/components/_volumecontrolbutton.scss

```diff
@@ -8,7 +8,7 @@
   position: relative;
 
   .#{$prefix}-ui-volumeslider {
-    @include animate-slide-in-from-bottom(6em, $animation-duration / 2);
+    //@include animate-slide-in-from-bottom(6em, $animation-duration / 2);
```

Comment:
Please remove this line (and the following empty line) entirely and add an additional entry into the `Changed` section of the `CHANGELOG`:

"Removed `VolumeControlButton`'s `VolumeSlider` slide-in animation in the legacy skin to fix the slider knob at 100% bug"

(This is a change in the skin's behavior and it's not obvious from the bugfix entry)